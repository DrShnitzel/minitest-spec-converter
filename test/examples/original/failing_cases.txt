class FailingCasesTest < Minitest::Test
  def test_hash_vs_named_params
    assert_equal({a: 1, b: 2}, some_method(a: 1, b: 2))
  end
end
