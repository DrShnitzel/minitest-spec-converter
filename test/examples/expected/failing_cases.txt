class FailingCasesTest < Minitest::Test
  it "hash vs named params" do
    _(some_method(a: 1, b: 2)).must_equal({a: 1, b: 2})
  end
end
